src/app/pagamentoplanos/component/pagamentoplanos.component.ts

```diff
@@ -1,4 +1,6 @@
 import { Component, OnInit } from '@angular/core';
+import { ExternalService } from '../service/external.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8398929)
```suggestion
import { ExternalService } from "../service/external.service";
```